content/docs/operating/configuration.md

```diff
@@ -6,66 +6,189 @@ nav_icon: sliders
 # Configuration
 
 Prometheus is configured via command-line flags and a configuration file. While
-the command-line flags configure general system parameters (such as storage
+the command-line flags configure immutable system parameters (such as storage
 locations, amount of data to keep on disk and in memory, etc.), the
 configuration file defines everything related to scraping [jobs and their
 instances](/docs/concepts/jobs_instances/), as well as which [rule files to
 load](/docs/querying/rules/#configuring-rules).
 
-To view all available command-line flags, run Prometheus with the `-h` option.
+To view all available command-line flags, run `prometheus -h`.
+
+
+## Configuration file
 
 To specify which configuration file to load, use the `-config.file` flag.
-A configuration file is written in ASCII protocol buffer form, and every
-available option is explained in detail in the protocol buffer schema
-definition at
-https://github.com/prometheus/prometheus/blob/master/config/config.proto.
+The configuration file (including rule files) can be reloaded at runtime by 
+sending SIGHUP to the Prometheus process.
+
+The file is written in [YAML format](http://en.wikipedia.org/wiki/YAML),
+defined by the scheme described below.
+Brackets indicate that a parameter is optional. For non-list parameters the
+value is set to the specified default.
+
+Generic placeholders are defined as follows:
+
+* `<duration>`: a duration matching the regular expression `[0-9]+[smhdwy]`
+* `<labelname>`: a string matching the regular expression `[a-zA-Z_][a-zA-Z0-9_]*`
+* `<labelvalue>`: a string of unicode characters
+* `<filename>`: a valid path in the current working directory
+
+The other placeholders are specified separately.
+
+A valid example file can be found [here](https://github.com/prometheus/prometheus/blob/fabxc/sd_yamlcfg/config/testdata/conf.good.yml).
```

Comment:
Yes, this has to be adjusted once the changes are in master. There will also be some additions before we v0.14 with which those docs become public and valid.

I should have merged this PR into a separate branch in the docs repo to avoid confusion - my bad.

If you want to take a look, this is the example config in the working branch: https://github.com/prometheus/prometheus/blob/fabxc/servdisc/config/testdata/conf.good.yml